main.go

```diff
@@ -1,22 +1,20 @@
 package main
 
 import (
-	"os"
-
+	"fmt"
 	"github.com/appuio/image-cleanup/cmd"
+	log "github.com/sirupsen/logrus"
 )
 
 var (
-	version string
-	commit  string
-	date    string
+	version = "unknown"
+	commit  = "dirty"
+	date    = "today"
 )
 
 func main() {
-
-	command := cmd.NewCleanupCommand(cmd.Build{Version: version, Commit: commit, Date: date})
-
-	if err := command.Execute(); err != nil {
-		os.Exit(1)
+	cmd.SetVersion(fmt.Sprintf("%s, commit %s, date %s", version, commit, date))
+	if err := cmd.Execute(); err != nil {
+		log.WithError(err).Fatal("Command aborted.")
```

Comment:
When I approach the CLI as a noob my first impression is that the user feedback is intimidating. The CLI fails hard when an argument is not or not correctly provided.

I would suggest a more typical approach, which is simply showing the usage, like e.g. the [hugo CLI client](https://github.com/gohugoio/hugo/blob/master/main.go#L25-L31) does it.